lib/resolvers/invitation.js

```diff
@@ -27,7 +27,14 @@ module.exports = ({ models, jwt, emailer }) => ({ action, data }, transaction) =
             });
           })
           .then(() => Establishment.query(transaction).findById(data.establishmentId))
-          .then(e => emailer.sendEmail({ ...data, token, establishmentName: e.name }))
+          .then(e => {
+            return Promise.resolve()
+              .then(() => emailer.sendEmail({ ...data, token, establishmentName: e.name }))
+              .catch(error => {
+                console.log(`failed to send invite email to "${args.email}" for "${e.name}"`);
+                console.log('error was: ', error);
+              });
```

Comment:
I'd move this `catch` into the emailer module so it's caught in all instances. Make the code here_much_ cleaner too.